### PR TITLE
PAR - Thomas & Jad - done!

### PR DESCRIPTION
**Pull request by jadch**

### CODE REVIEW

File: starter-code/app.js [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+app.get('/', (req, res, next) => {
+  res.render('index')
+});
+
+app.get('/random', (req, res, next) => {
+  let data = {
+    test: 'woo'
```
Comment (yacinehmito) [inline starter-code/app.js:20]: Clean up your code guys ;)

File: starter-code/app.js [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+});
+
+app.get('/categories', (req, res, next) => {
+  let data = {
+    test: 'woo'
+  }
+
+  client.getJokeCategories().then((response) => {
+    data['categories'] = response
+    console.log('***** DATA *****', data)
```
Comment (yacinehmito) [inline starter-code/app.js:38]: Really.

File: starter-code/app.js [hunk 3]
```diff
@@ -1,5 +1,73 @@
...
+    res.render('categories', data)
+  }).catch((error) => {
+    throw error
+  })
+});
+
+app.get('/category', (req, res, next) => {
```
Comment (yacinehmito) [inline starter-code/app.js:45]: You're supposed to use the `/categories` route.

**Review by yacinehmito — APPROVED** [review]
Very good job! 🎉 
It could be ever better with layouts, but other than that great work. 💯 

Also, you may want to take the habit of rereading your code before committing. There's some unneeded dead code and `console.log` that could have been removed.

File: starter-code/app.js (hunk 1 above)
Comment (liloow) [reply starter-code/app.js:20]: Haha

File: starter-code/app.js (hunk 3 above)
Comment (liloow) [reply starter-code/app.js:45]: We thought we got them all :disappointed:

**Pull request closed without merging**